apps/web/src/composables/wallet.ts

```diff
@@ -53,7 +53,11 @@ export default function useWallet() {
     selectedAccount.value = address
   }
 
-  async function connectWallet(provider: ProviderString) {
+  const setSelectedToken = (token: TokenString) => {
+    selectedToken.value = token
+  }
+
+  async function connectWallet(provider: ProviderString, token?: TokenString) {
```

Comment:
@ccali11 what if we turn this into an options object (if you have time a ConnectWalletOptions.ts interface) and change 'token' to 'currency'?

Reply:
You got it!